package.json

```
@@ -24,7 +24,7 @@
     "test:unit": "cross-env NODE_PATH=. TS_NODE_PROJECT=app/server/tsconfig.json NODE_ENV=test nyc mocha --exit --recursive --require jsdom-global/register \"test/unit/*.ts\" \"test/unit/**/*.ts\"",
     "test:unit-server": "cross-env NODE_PATH=. TS_NODE_PROJECT=app/server/tsconfig.json NODE_ENV=test mocha test/unit/*.ts test/unit/**/*.ts",
     "test:unit-client": "cross-env NODE_PATH=. TS_NODE_PROJECT=app/client/tsconfig.json NODE_ENV=test mocha --require jsdom-global/register test/unit/client/**/*.ts",
-    "test:browser": "NODE_PATH=. TS_NODE_PROJECT=app/server/tsconfig.json NODE_ENV=test mocha test/browser/**/*.ts --exit",
+    "test:browser": "NODE_PATH=. TS_NODE_PROJECT=app/server/tsconfig.json NODE_ENV=test mocha test/browser/**/*.ts --colors --reporter mochawesome  --reporter-options reportDir=functional-output/reports/functional --exit",
```

I believe you can add `inlineAssets=true` to make the report self-contained so that it doesn't look odd on Jenkins and can be easily downloaded
```suggestion
    "test:browser": "NODE_PATH=. TS_NODE_PROJECT=app/server/tsconfig.json NODE_ENV=test mocha test/browser/**/*.ts --colors --reporter mochawesome  --reporter-options reportDir=functional-output/reports/functional,inlineAssets=true --exit",
```

---

@Rob-Doherty  can you explain more  ?